dashboard/src/modules/components/BackgroundComponent/index.less

```diff
@@ -1,6 +1,6 @@
 .background-image {
   background-image: linear-gradient(rgba(0, 0, 0, 0.527), rgba(0, 0, 0, 0.5)),
-    url("assets/images/pfbg_576@2x.jpg");
+    url("../../../assets/images/pfbg_576@2x.jpg");
```

Comment:
Is there a configuration file that we can use to specify the root-path, so that we can avoid using long sequences of double-dots?

Reply:
Done!